### DISCUSSION TITLE
Webpage SDK: Create Chart by baseURL

### DISCUSSION THREAD

**Discussion by gabac** [TelemetryDeck/Viewer#330] [Q&A] [answered]
Hi community

I'm looking for a way to create a chart based on the "base" url without the path part?

<img width="367" alt="image" src="https://github.com/TelemetryDeck/Issues/assets/50061/2bac01de-d67c-44b8-9bed-dec381baee6b">

Any help would be appreciated.

**Comment by winsmith** [comment, marked as answer]
I just pushed an update to the ingest API that will add the "path" and "host" properties to signals sent from the Web SDK. So as of today, you can use "host" as your breakdown and should get just what you want.

**Reply by gabac** [reply]
That works as expected. Thanks!
